### PR TITLE
itR-10: feat: do switching themes

### PR DESCRIPTION
**Pull request by MilaKolasava**
image
image

### CODE REVIEW

File: src/components/Footer/Footer.tsx [hunk 1]
```diff
@@ -3,17 +3,34 @@ import "./Footer.css";
...
+interface FootersProps {
+  setIsDarkThemeOff: React.Dispatch<React.SetStateAction<boolean>>;
+  isDarkThemeOff: boolean;
+}
+
+function Footer(props: FootersProps) {
   return (
-    <nav className="wrapper" id="footer" data-testid="footer-wrapper">
+    <nav
+      className={`wrapper wrapper-${props.isDarkThemeOff ? "dark" : "light"}`}
```
Comment (glebpigulevsky) [inline src/components/Footer/Footer.tsx]: I suggest to create utility function because here is a lot of repeatable code (14, 24, 31 strings)

**Review by glebpigulevsky — CHANGES_REQUESTED** [review]
tests

File: src/components/Footer/Footer.tsx (hunk 1 above)
Comment (MilaKolasava) [reply src/components/Footer/Footer.tsx]: good idea! 
done

File: src/components/Toggle/Toggle.tsx [hunk 2]
```diff
@@ -1,32 +1,35 @@
...
         className={`toggle-dot dot-${
-          isDarkThemeOff ? "right" : "left"
+          props.isDarkThemeOff ? "right" : "left"
         }-position`}
         data-testid="toggle-dot"
       />
       <div
         className={`toggle-switch toggle-switch-${
-          isDarkThemeOff ? "left" : "right"
+          props.isDarkThemeOff ? "left" : "right"
```
Comment (glebpigulevsky) [inline src/components/Toggle/Toggle.tsx:29]: getColorTheme? and so on

**Review by glebpigulevsky — APPROVED** [review]
good job

**Pull request merged**